src/main/java/uk/gov/MintApplication.java

```diff
@@ -52,9 +60,19 @@ public void run(MintConfiguration configuration, Environment environment) throws
         EntryValidator entryValidator = new EntryValidator(registersConfiguration, fieldsConfiguration);
 
         LoadHandler loadHandler = new LoadHandler(configuration.getRegister(), jdbi.onDemand(EntriesUpdateDAO.class), entryValidator);
+        final Client client = new JerseyClientBuilder(environment)
+                .using(configuration.getJerseyClientConfiguration())
+                .build(getName());
+        if(client == null && StringUtils.isNotBlank(configuration.getCTServer())) {
```

Comment:
Is it an example of defensive coding? 

we should create client object only if ctserver configuration is present. validity of jersey client configuration is already confirmed by dropwizard.